src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerFileReference.cs

```diff
@@ -465,8 +465,7 @@ public bool Equals(AnalyzerReference other)
         {
             if (other != null)
             {
-                return other.Display == this.Display &&
-                       other.FullPath == this.FullPath;
+                return other is { Display: this.Display, FullPath: this.FullPath };
```

Comment:
>this.Display [](start = 43, length = 12)

Not a valid pattern.

Reply:
➡️ Treating this as a duplicate of https://github.com/dotnet/roslyn/pull/39738#discussion_r344405091